### DISCUSSION TITLE
Lifetime license

### DISCUSSION THREAD

**Discussion by vietxtom** [aclap-dev/video-downloadhelper#30] [License & Purchase requests]
### What's your issue?

I need a new lifetime license because  my app is now set  to License not set.

### The 10 first digits (only 10) of your license number or email address

tomtrongnguyen@gmail.com

### I confirm that I didn't not paste my full license number

- [X] I confirm

**Comment by Salmonovich** [comment]
Hi, I have the same problem.

**Reply by jcfrog** [reply]
please read this: https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#license-problem
